docs/standard/data/sqlite/TOC.yml

```diff
@@ -54,6 +54,6 @@
     - name: Xamarin
       href: xamarin.md
 - name: API reference >>
-  href: /dotnet/api/?view=msdata-sqlite-3.0
+  href: /dotnet/api/?view=msdata-sqlite-3.1.0
```

Comment:
```suggestion
  href: /dotnet/api/?view=msdata-sqlite-3.1
```

Reply:
Scratch that, it looks like our attempt to rename the monikers failed and that's probably why you're submitting this PR. I'm going to revert the renames for 1.0-3.0. Using `msdata-sqlite-3.1.0` is correct for now.